setup.py

```diff
@@ -1,42 +1,151 @@
+import distutils
 import glob
 import os
+import shlex
 import setuptools
 import setuptools.command.build_ext
+import setuptools.command.develop
+import setuptools.command.install
 import setuptools.dist
 import sys
 import setuptools.command.install
 import cellprofiler.utilities.version
 
+# Recipe needed to get real distutils if virtualenv.
```

Comment:
Is this needed? I don’t think we’re using distutils anywhere anymore. If we are, we should update to setuptools since distutils was deprecated a few years ago.